library/core/src/prelude/mod.rs

```diff
@@ -1,5 +1,41 @@
 //! The libcore prelude
+//!
+//! This module is intended for users of libcore which do not link to libstd as
+//! well. This module is imported by default when `#![no_std]` is used in the
+//! same manner as the standard library's prelude.
 
 #![stable(feature = "core_prelude", since = "1.4.0")]
 
 pub mod v1;
+
+/// The 2015 version of the core prelude.
+///
+/// See the [module-level documentation](../index.html) for more.
```

Comment:
```suggestion
/// See the [module-level documentation](self) for more.
```
Note this is slightly tricky because it depends on rustdoc resolving relative to the containing module for `///`, you may want to double check (or just use `crate::prelude`) just in case.